library/ssl_tls13_generic.c

```diff
@@ -1848,58 +1785,130 @@ static int ssl_certificate_verify_write( mbedtls_ssl_context* ssl,
                                          size_t* olen )
 {
     int ret;
-    const mbedtls_ssl_ciphersuite_t* ciphersuite_info = ssl->handshake->ciphersuite_info;
     size_t n = 0, offset = 0;
-
-    unsigned char hash[MBEDTLS_MD_MAX_SIZE];
-    unsigned char* hash_start = hash;
-    unsigned int hashlen;
-
-    /* TODO: Add bounds checks! Only then remove the next line. */
-    ( (void ) buflen );
-
-    /*
-     * Make a signature of the handshake transcript
-     */
-    ret = ssl->handshake->calc_verify( ssl, hash, ssl->conf->endpoint );
-
-    if( ret != 0 )
-    {
-        MBEDTLS_SSL_DEBUG_RET( 1, "calc_verify", ret );
-        return( ret );
+    unsigned char verify_buffer[ 64 + 33 + 1 + MBEDTLS_MD_MAX_SIZE ];
```

Comment:
As above: It would be helpful to introduce a few macro constants here to explain where those numbers come from.

Reply:
Used a macro here instead